lib/test/bull.explorer.spec.ts

```diff
@@ -1,18 +1,26 @@
 import { InstanceWrapper } from '@nestjs/core/injector/instance-wrapper';
-import { Test } from '@nestjs/testing';
+import { Test, TestingModule } from '@nestjs/testing';
 import { BullExplorer } from '../bull.explorer';
 import { BullModule } from '../bull.module';
 import { getQueueToken } from '../utils';
 
-describe('bullExplorer', () => {
+describe('BullExplorer', () => {
   let bullExplorer: BullExplorer;
-  beforeEach(async () => {
-    const moduleRef = await Test.createTestingModule({
-      imports: [BullModule.registerQueue({ name: 'test' })],
+  let moduleRef: TestingModule;
+
+  beforeAll(async () => {
+    moduleRef = await Test.createTestingModule({
+      imports: [
+        BullModule.registerQueue({ name: 'test', redis: { port: 6380 } }),
+      ],
     }).compile();
 
     bullExplorer = moduleRef.get(BullExplorer);
   });
+  afterAll(async () => {
+    await moduleRef.get(getQueueToken('test')).close();
```

Comment:
```suggestion
    await moduleRef.get(getQueueToken('test')).close();
```
I believe that this line shouldn't be needed